File: Python/11720_숫자의합.py
if __name__ == "__main__":
    N = int(input())
    S = input()
    
    S_sum = 0
    for i in S: #S를 문자로 받아서 더하기(더할 때 숫자로 변환)
        S_sum += int(i)
        
    print(S_sum)

File: C++/11720_숫자의합.cpp
#include <iostream>
#include <stdio.h>
using namespace std;

int main(){
    int N;
    cin >> N;
    
    int S, S_sum = 0;
    for(int i = 0; i < N; i++){ //한 자리 씩 받아서 더하기
        scanf("%1d",&S);
        S_sum += S;
    }
    
    cout << S_sum;
    
    return 0;
}